package.json

```diff
@@ -55,6 +55,7 @@
   "dependencies": {
     "@babel/runtime": "^7.12.5",
     "@types/sdp-transform": "^2.4.5",
+    "@types/uuid": "7",
```

Comment:
This should go into dev deps `@types`

Reply:
https://github.com/matrix-org/matrix-js-sdk/pull/2927 fixes it for the naughty stragglers